boutiques_descriptors/explore-asl.json

```diff
@@ -3,17 +3,19 @@
+    "author": "H. Mutsaerts et al.",
+    "url": "https://exploreasl.github.io/Documentation/1.9.0/",
+    "command-line": "run_xasl.py --mode BIDS [INPUT_FOLDER] [OUTPUT_FOLDER] [ImportModules_DEFACE] [ProcessModules_STRUCTURAL] [ProcessModules_ASL] [X__SESSIONS] [X__SESSION__OPTIONS] [X__DATASET__SUBJECTREGEXP] [X__DATASET__EXCLUSION] [X__DATASET__FORCEINCLUSIONLIST] [X__MODULES__ASL__M0_CONVENTIONALPROCESSING] [X__MODULES__ASL__M0_GMSCALEFACTOR] [X__MODULES__ASL__M0POSITIONINASL4D] [X__MODULES__ASL__DUMMYSCANPOSITIONINASL4D] [X__Q__M0] [X__Q__BACKGROUNDSUPPRESSIONNUMBERPULSES] [X__Q__BACKGROUNDSUPPRESSIONPULSETIME] [X__Q__PRESATURATIONTIME] [X__Q__READOUTDIM] [X__Q__VENDOR] [X__Q__SEQUENCE] [X__Q__LABELINGTYPE] [X__Q__INITIAL_PLD] [X__Q__LABELINGDURATION] [X__Q__SLICEREADOUTTIME] [X__Q__LAMBDA] [X__Q__T2ART] [X__Q__BLOODT1] [X__Q__TISSUET1] [X__Q__NCOMPARTMENTS] [X__Q__APPLYQUANTIFICATION] [X__Q__SAVECBF4D] [X__MODULES__ASL__MOTIONCORRECTION] [X__MODULES__ASL__SPIKEREMOVALTHRESHOLD] [X__MODULES__ASL__BREGISTRATIONCONTRAST] [X__MODULES__ASL__BAFFINEREGISTRATION] [X__MODULES__ASL__BDCTREGISTRATION] [X__MODULES__ASL__BREGISTERM02ASL] [X__MODULES__ASL__BUSEMNIASDUMMYSTRUCTURAL] [X__MODULES__ASL__BPVCNATIVESPACE] [X__MODULES__ASL__PVCNATIVESPACEKERNEL] [X__MODULES__ASL__BPVCGAUSSIANMM] [X__MODULES__ASL__BMAKENIFTI4DICOM] [X__MODULES__BRUNLONGREG] [X__MODULES__BRUNDARTEL] [X__MODULES__STRUCTURAL__BSEGMENTSPM12] [X__MODULES__STRUCTURAL__BHAMMERSCAT12] [X__MODULES__STRUCTURAL__BFIXRESOLUTION] [X__SETTINGS__QUALITY] [X__SETTINGS__DELETETEMP] [X__SETTINGS__SKIPIFNOFLAIR] [X__SETTINGS__SKIPIFNOASL] [X__SETTINGS__SKIPIFNOM0] [X__SETTINGS__STOPAFTERERRORS] [X__SETTINGS__BLESIONFILLING] [X__SETTINGS__BAUTOACPC] [X__S__BMASKING] [X__S__ATLASES]",
     "container-image": {
-        "image": "dvalev/xasl_cbrain:1.1.2",
+        "image": "dvalev/xasl_cbrain:1.2.0",
         "index": "docker://",
         "type": "singularity"
     },
     "inputs": [
         {
             "id": "input_folder",
             "name": "Input Folder",
-            "description": "Data folder to be processed. Can be single-subject or multi-subject, in DICOM or BIDS (NIfTI) format.\nNOTE: For the Structural and ASL process modules, a CBCSV can be used to process multiple subjects. For the Population module, all subjects must be included in one file collection.",
+            "description": "BIDS Subject folder to be processed. A CBCSV file can be used to process multiple subjects in parallel.",
```

Comment:
I don't like this description, it makes de descriptor specific to CBRAIN.

The input should just be described as a BIDS datasets, because that's what the tool wants. Everything specific to CBRAIN is outside the scope of the tool's scientific domain. In fact, CBRAIN will automatically add a note about using a BIDS subject instead of a BIDS datasets, and CBRAIN will also generate a note about using CBCSV in its help page.

Reply:
The point is that you should be able ot use this tool using bosh directly on the command line, and in CBRAIN, the tool's input are modified automatically by the framework to adapt to the new environement (CBRAIN).

Reply:
> I don't understand your argument, Darius. If you use bosh on a local machine, you have control of the file system including the files used by the tool. You need a dataset_description? You just `cp` it in place, no?

You're entirely right, it works fine if I just pass a dataset rather than a subject as input. I'll just clean up the wrapper to only ever expect a dataset (could take either dataset or subject + description before).

Reply:
Addressed both points [[8fc14ee](https://github.com/aces/cbrain-plugins-neuro/pull/263/commits/8fc14eef7221ca2e06164377ce19f2d1635b08f1)]